app/models/book_request.rb

```diff
@@ -0,0 +1,11 @@
+class BookRequest < ApplicationRecord
+  belongs_to :user
```

Comment:
Write test.

Reply:
Ready in `spec/models/book_request_spec.rb`